manifests/install.pp

```diff
@@ -3,7 +3,11 @@
 # Class to install xtrabackup from packages
 #
 class xtrabackup::install {
-  if $::lsbdistcodename != 'lucid' {
+  if $xtrabackup::compatibility_version == 'mysql80' {
+    package {'percona-xtrabackup-80':
+      ensure => $xtrabackup::version
+    }
+  } elsif $::lsbdistcodename != 'lucid' {
```

Comment:
we can also remove the lucid check here and just make this `else`

Reply:
I'm not sure that this corner cases is no longer in play, @VinaySagarGonabavi do you have further insight into this?

Reply:
There are no hosts with the ubuntu version lucid anymore. I don't think this is relevant anymore:
```
$ inv find -F lsbdistcodename=lucid

gonabavi@dev60-uswest1adevc:~/pg 2020-11-10 14:41:09

$
```

Reply:
fixed per conversation in slack